cmd/minishift/cmd/delete.go

```diff
@@ -40,8 +40,6 @@ func runDelete(cmd *cobra.Command, args []string) {
 	api := libmachine.NewClient(constants.Minipath, constants.MakeMiniPath("certs"))
 	defer api.Close()
 
-	setSubcriptionManagerParameters()
-
```

Comment:
> yes we don't have username/password flag for stop/delete any more

:-( This is really not how I thought it would turn out in the end. It is after all just tailored now for RHEL subscription.

> but that doesn't effect our reel regitratrar

Sure, I get that

> once we have #487 implemented then take that details from there and use it for delete/stop. We can create a issue for same just to track.

Ok, so there is a follow up plan, at least in terms of tracking it. Even though I like the idea of doing this via storing the credentials in the config file, it has the "issue" of storing the password plain text. 

BTW, surely it would be possible to have the same flag name on multiple sub-commands, right? This is the easier way out for us atm, but technically it should be possible. I cannot imagine that Viper has a constraint that all flags need to be unique per command.